ingestion-beam/src/main/java/com/mozilla/telemetry/options/OutputType.java

```diff
@@ -223,7 +223,7 @@ public EmptyErrors(PTransform<PCollection<PubsubMessage>, ? extends POutput> inn
     @Override
     public ResultWithErrors<? extends POutput> expand(PCollection<PubsubMessage> input) {
       return ResultWithErrors.of(input.apply(inner),
-          input.getPipeline().apply(Create.empty(PubsubMessageWithAttributesCoder.of())));
+          input.getPipeline().apply("Empty error collection", Create.empty(PubsubMessageWithAttributesCoder.of())));
```

Comment:
Small unrelated change. It's difficult to understand in the Dataflow UI what this Create.Values instance means.